src/lib/controllers/config-store/index.d.ts

```diff
@@ -0,0 +1,3 @@
+export interface ConfigMap {
```

Comment:
Give this a separate name rather than using `index.d.ts`, as that implies it defines *all* the external types for the `config-store` classes, which it doesn't.